README.md

```diff
@@ -6,4 +6,30 @@ A bakery for use by Spinnaker to produce machine images.
 
 It presently supports producing Google Compute Engine images and AWS amis. It relies on packer and can be easily extended to support additional platforms.
 
-It exposes a REST api which can be experimented with via the Swagger UI: http://localhost:8087/swagger/index.html
+It exposes a REST api which can be experimented with via the Swagger UI: http://localhost:8087/swagger-ui.html
+
+# Developing rosco
+
+Need to run rosco locally for development? Here's what you need to setup and run:
+
+## Environment Setup
+. git clone git@github.com:spinnaker/rosco.git

```

Comment:
I'd prefer to have the spinnaker/spinnaker repo cloned as a sibling, and then have the docker-compose command use the config from here: https://github.com/spinnaker/spinnaker/blob/master/experimental/docker-compose/docker-compose.yml#L90-L95

Just trying to avoid config proliferation.